src/python/pants/backend/go/go_sources/load_go_binary.py

```diff
@@ -30,6 +39,49 @@ def debug_hint(self) -> str:
         return self.output_name
 
 
+@dataclass(frozen=True)
+class NaiveBuildGoPackageRequestForStdlibPackageRequest:
```

Comment:
I assume the naivety is due to the bootstrapping problem?

Reply:
Exactly. These rules are used to build the package analyzer. so we have no package analyzer available yet to run.

I'll add a comment explaining that.